pkg/cmd/cli/cmd/newapp.go

```diff
@@ -101,6 +101,7 @@ func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer)
 	cmd.Flags().StringVar(&config.Name, "name", "", "Set name to use for generated application artifacts")
 	cmd.Flags().StringVar(&config.Strategy, "strategy", "", "Specify the build strategy to use if you don't want to detect (docker|source).")
 	cmd.Flags().StringP("labels", "l", "", "Label to set in all resources for this application.")
+	cmd.Flags().BoolVar(&config.Insecure, "insecure-registry", false, "If true indicates that images specifying a registry reside in an insecure registry")
```

Comment:
The sentence is a bit fragmented - "If true, indicates that the referenced Docker images are on insecure registries and should bypass certificate checking"